config/clusters/2i2c/ohw.values.yaml

```diff
@@ -73,6 +73,8 @@ basehub:
       2i2c:
         add_staff_user_ids_to_admin_users: true
         add_staff_user_ids_of_type: "github"
+      hubTemplateRepoDetails:
+        repo_branch: "2i2c-ohw"
       homepage:
         templateVars:
           org:
```

Comment:
There is already `custom.homepage`, and this is directly related to that right?

If so, should we just let this be nested under `custom.homepage` instead?